tests/neo.UnitTests/SmartContract/UT_JsonSerializer.cs

```diff
@@ -32,9 +32,6 @@ public void JsonTest_WrongJson()
 
             json = @"{""length"":99999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999}";
             Assert.ThrowsException<FormatException>(() => JObject.Parse(json));
-
```

Comment:
Why we need to remove some UT?

Reply:
It shouldn't throw exceptions. The old UT is incorrect.